src/package-operations.ts

```diff
@@ -381,3 +426,18 @@ function isMonorepoUpdateSpecification(
     'synchronizeVersions' in specification
   );
 }
+
+/**
+ * Given a path string, returns the last two segments of the path, without
+ * leading or trailing slashes.
+ *
+ * @param absolutePath - The absolute path to truncate.
+ * @returns The truncated path string.
+ */
+function getTruncatedPath(absolutePath: string): string {
+  return absolutePath
+    .split('/')
```

Comment:
Nit: we can use `path.sep` here for better windows support

Reply:
Done in b5096e0